### PR TITLE
Fullness detection when crafting

### PR DESCRIPTION
**Pull request by InusualZ**

### CODE REVIEW

File: src/main/java/refinedstorage/tile/grid/TileGrid.java [hunk 1]
```diff
@@ -257,7 +254,10 @@ public void onCrafted(EntityPlayer player) {
             if (i < remainder.length && remainder[i] != null) {
                 if (slot != null && slot.stackSize > 1) {
                     if (!player.inventory.addItemStackToInventory(remainder[i].copy())) {
-                        network.insertItem(remainder[i].copy(), remainder[i].stackSize, false);
+                        ItemStack remain = network.insertItem(remainder[i].copy(), remainder[i].stackSize, false);
```
Comment (raoulvdberge) [inline src/main/java/refinedstorage/tile/grid/TileGrid.java]: varname should be remainder not remain

File: src/main/java/refinedstorage/tile/grid/TileGrid.java [hunk 2]
```diff
@@ -257,7 +254,10 @@ public void onCrafted(EntityPlayer player) {
...
+                        if (remain != null) {
+                            InventoryHelper.spawnItemStack(player.worldObj, pos.getX(), pos.getY(), pos.getZ(), remain);
```
Comment (raoulvdberge) [inline src/main/java/refinedstorage/tile/grid/TileGrid.java]: should it spawn at the player pos and not grid pos?

File: src/main/java/refinedstorage/tile/grid/TileGrid.java (hunk 1 above)
Comment (InusualZ) [inline src/main/java/refinedstorage/tile/grid/TileGrid.java]: Look up. ItemStack Array called remainder

Comment (way2muchnoise) [inline src/main/java/refinedstorage/tile/grid/TileGrid.java]: remainderStack?

**Pull request merged**